src/scripts/check.sh

```diff
@@ -33,12 +33,13 @@ Check_for_shellcheck() {
 
 Run_ShellCheck() {
     SC_PARAM_PATTERN="${SC_PARAM_PATTERN:-"*.sh"}"
-    find "$SC_PARAM_DIR" ! -name "$(printf "*\n*")" -name "$SC_PARAM_PATTERN" > tmp
+    sc_input_files=/tmp/sc-input-files
```

Comment:
Should be capitalised to align with the other variables in this script.

Reply:
Fixed.